core/src/main/java/org/elasticsearch/rest/action/support/RestTable.java

```diff
@@ -322,15 +322,15 @@ private static String renderValue(RestRequest request, Object value) {
             String resolution = request.param("size");
             if ("b".equals(resolution)) {
                 return Long.toString(v.singles());
-            } else if ("k".equals(resolution)) {
+            } else if ("k".equals(resolution) || "kb".equals(resolution)) {
```

Comment:
afaics kb is not supported for SizeValue. it is "k" or "K"? Same for the other entries.